Sources/MongoSwift/ChangeStreamDocument.swift

```diff
@@ -0,0 +1,76 @@
+/// An `UpdateDescription` containing fields that will be present in the change stream document for
```

Comment:
indentation also seems off in this file. `UpdateDescription` and `OperationType`'s properties are only indented three spaces